website/integrations/miscellaneous/open-webui/index.md

```diff
@@ -42,16 +42,27 @@ To support the integration of Open WebUI with authentik, you need to create an a
 
 ## Open WebUI configuration
 
-You need to set the environment variables in the appropriate place based on your deployment method—either in the .env file or as Docker environment variables. The values themselves remain the same, only their location differs.
+To configure authentik with Open WebUI, you must add the following environment variables to your OpenWebUI deployment:
```

Comment:
Please add a sentence right after line 38, something like " - **Encryption Key**: be sure to leave the field empty."